tests/queries/0_stateless/02353_translate.sql

```diff
@@ -10,4 +10,10 @@ SELECT translate('abc', '', '');
 SELECT translateUTF8('abc', '', '');
 
 SELECT translate('abc', 'Ááéíóúôè', 'aaeiouoe'); -- { serverError BAD_ARGUMENTS }
-SELECT translateUTF8('abc', 'efg', ''); -- { serverError BAD_ARGUMENTS }
+SELECT translateUTF8('abc', 'efg', '');
+
+SELECT translateUTF8('中文内码', '中文', '');
+SELECT translate('aAbBcC', 'abc', '12');
+
+SELECT translate('aAbBcC', 'abc', '1235');
+SELECT translate('aAbBcC', '中文内码', '12'); -- { serverError BAD_ARGUMENTS }
```

Comment:
Could you also please add a test like:

SELECT translate('abc', 'abc', '');

Reply:
I think you forgot to add the test to commit

Reply:
sorry，done now